src/utils/draftmode.ts

```diff
@@ -11,3 +11,13 @@ export function getDraftModeInfo(): DraftModeInfo {
   const perspective = isDraftMode ? "previewDrafts" : "published";
   return { isDraftMode, perspective };
 }
+
+export function checkDraftDataIfDevelopment(data: unknown) {
```

Comment:
since we're validating the data how about giving it a more specific name? 

```suggestion
export function validateDraftDataInDevelopment(data: unknown): void {
```

I'd also add void the explicit return type, just to clarify that the function is not expected to return any value.